### ISSUE TITLE
AEP comments 03-12

### ISSUE THREAD

**Issue by mcsiple** [mcsiple/mmrefpoints#36]
03_get_rf
- [x] Can you look at my edits the description stuff?
- [x] P0 and N0 don’t render correctly in help
- [x] Check E_in is within bounds.
- [x] You can simplify the code given R0=1.
- [x] I added one call to npr (should speed things up)

06_get_f
- [x] - Add bounds on f.start (0 to 1) and lambdamax (>1)
- [x] - See my edits  (tify up)
- [x] - If f.start actually used?

07_ce
- [x] Removed lambdamax from the documentation (not used)
- [x] The example had an error, which I fixed
- [x] Add added bounds on E

08_get_dc
- [x] Implement edits to the documentation as suggested
- [x] Dropped references to lambdamax and K1Plus (not used)

09_find_msyr
- [x] Removed reference to K1Plus because it is not used.
- [x] Why do you pass N0 when it computed in the function. Simpler not to pass it.

10_get_mnpl
- [x] See my edits to simplify the code
- [x] I removed references to K1+plus and edited a bit

11_Get_dc
- [x] I removed references to K1+plus and edited a bit
- [x] Is fmax used anywhere? I don’t think so because fmax is calculated in get_mnpl

12_calc_z
- [x] Add bounds on MNPL
- [x] I removed references to K1+plus and edited a bit
- [x] Is fmax used anywhere? I don’t think so because fmax is calculated in get_mnpl

**Comment by mcsiple** [comment]
03: I'm going to leave the `N0` and `P0` now. They don't render correctly because the RStudio HTML version of the help [won't render latex expressions](https://stackoverflow.com/questions/37397607/can-r-help-manuals-have-latex-math-in-them) (annoying) but they WILL be formatted correctly in the pdf doc. I think there is a solution here but it seems kind of tricky/involves editing the .Rd file. 3812ebcf

06: `f.start` is for used for getting the starting F in the solver on line 56. Fixed with commit 516e98e0

07: Need to go back and check bc no file from AEP but I did make all the changes in this list so my version should be good. Checked the example again. Commits 2194b5f8 and 224181ad

08: Doc edits 82d73c3e
